### PR TITLE
feat: ✨ Updated wizard styles for the stack

### PR DESCRIPTION
**Pull request by GoOz**
Some additions and adjustments for the stack's wizard

### CODE REVIEW

**Comment by GoOz** [comment]
[(𝐒) [LOGIN] Amélioration de la page de mot de passe](https://trello.com/c/hsuOA8Ub/363-%F0%9D%90%92-login-am%C3%A9lioration-de-la-page-de-mot-de-passe)

File: stylus/components/forms.styl [hunk 1]
```diff
@@ -311,13 +311,13 @@ $checkbox
             box-shadow inset 0 0 0 rem(2) dodgerBlue
 
         &::after
-            background-image embedurl('../../assets/icons/ui/check.svg')
+            background-image embedurl('../../assets/icons/ui/check-white.svg')
```
Comment (CPatchane) [inline stylus/components/forms.styl]: Colorful icon again, we may need to put them on a separate icon folder one day to avoid use them with `<Icon />`

Comment (GoOz) [reply stylus/components/forms.styl]: Yup.
But isn't your incoming work on the SVG sprite specifically target a list of icon? It doesn't take them all, does it?

File: stylus/components/wizard.styl [hunk 2]
```diff
@@ -150,6 +208,15 @@ $wizard-input
...
 $wizard-notice
     margin     rem(8) 0
+
+    a
+        color dodgerBlue
+        text-decoration none
+
+        &:hover
+        &:focus
+            color scienceBlue
```
Comment (ptbrowne) [inline stylus/components/wizard.styl]: On devrait progressivement passer a des couleurs semantiques, capables de supporter du theme. Pour moi on devrait remplacer dodgerBlue par primaryColor et scienceBlue par primaryColor--dark.

**Pull request merged**